src/hooks/useFeedbackMessage/useMessageWrapper/useMessageWrapper.module.css

```diff
@@ -47,6 +47,14 @@
     &[class~=ant-message-move-up-leave] {
       animation-name: fadeOut;
     }
+
+    &[class~=mia-platform-message-move-up] {
```

Comment:
the mia-platform prefix is inserted by antd ConfigProvider and this selector should be built taking the relative constant, but this is not possible in a css file